addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -142,6 +142,9 @@ def in_eu(partner):
                 return True
             return False
 
+        def format_alphanumeric(any_string):
```

Comment:
any_string...  
```suggestion
        def format_alphanumeric(text_to_convert):
``` or something better @dbkosky